source/codegen/metadata/nidcpower/functions.py

```diff
@@ -1352,6 +1352,7 @@
       {
         'name': 'powerlineFrequency',
         'direction': 'in',
+        'enum': 'PowerLineFrequences',
```

Comment:
It should be `PowerLineFrequencies`. Also update in `enums.py` and `attributes.py` files.

Reply:
fixed